### PR TITLE
add searchtype options for lessthan/greaterthan/etc, default searchtype to 'equals' and searchfield to $key_column

### PR DESCRIPTION
**Pull request by joshrabinowitz**
adds new searchtypes. 
factors code defining searchtype descriptions and comparitors.

### CODE REVIEW

**Comment by joshrabinowitz** [comment]
I think this is ready for merge @bigpresh

File: lib/Dancer/Plugin/SimpleCRUD.pm [hunk 1]
```diff
@@ -966,24 +967,30 @@ sub _create_list_handler {
...
-        [ ne => "Does Not Equal" ],
+        [ e => { name=>"Equals", cmp=>"="} ],
+        [ c => { name=>"Contains", cmp=>"like"} ],
+        [ ne => { name=>"Does Not Equal", cmp=>"!="} ],
+        [ nc => { name=>"Does Not Contain", cmp=>"not like"} ],
+
+        [ lt => { name=>"Less Than", cmp=>"<"} ],
+        [ lte => { name=>"Less Than or Equal To", cmp=>"<="} ],
+        [ gt => { name=>"Greater Than", cmp=>">"} ],
+        [ gte => { name=>"Greater Than or Equal To", cmp=>">="} ],
```
Comment (bigpresh) [inline lib/Dancer/Plugin/SimpleCRUD.pm]: I think a little space around these would have made things clearer - e.g.

```perl
[ gte => { name => "Greater Than or Equal To", cmp => ">=" } ],
```

**Comment by bigpresh** [comment]
Other than minor stylistic comment, agree this is all good

**Comment by joshrabinowitz** [comment]
I think this is ready to go @bigpresh

**Comment by joshrabinowitz** [comment]
@bigpresh what do you want to see added to this before merge?

**Comment by bigpresh** [comment]
Right, have pushed 1.00_01 to CPAN, let's see what happens :)

**Pull request merged**